test/e2e/operator_test.go

```diff
@@ -1213,6 +1393,11 @@ func ensureCredentialsRequest(secret types.NamespacedName) error {
 				Effect:   "Allow",
 				Resource: "*",
 			},
+			{
+				Action:   []string{"tag:GetResources"},
+				Effect:   "Allow",
+				Resource: "*",
```

Comment:
I am not sure if we can restrict that. I think `Resource` takes the resource arn or `*` . Is there a way to limit that?

Reply:
You should be able to specify wildcard for some parts of the ARN too. Something like this should limit the tag querying to the LBs only:
```suggestion
				Resource: "arn:aws:elasticloadbalancing:*:*:loadbalancer/*",
```

Reply:
Ah, okay. Thanks for the suggestion. Updated the wildcard logic.

Reply:
CI is complaining with [AccessDeniedException](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_aws-load-balancer-operator/137/pull-ci-openshift-aws-load-balancer-operator-main-e2e-aws-proxy-operator/1863521573424599040#1:build-log.txt%3A653) due to this restriction. Should we revert back to `*` ?

Reply:
Access denied to which resource?

Reply:
I cannot find any RTG documentation which would explicitly state that `*` is required. However [ALBC used `*` when they added the RTG support](https://github.com/kubernetes-sigs/aws-load-balancer-controller/pull/3186/files/64f2095a418f26fe7c7caaf460db8fb2ded010cf#diff-e32786079cee1efe20f1c3df93601e08412e204542db0d95600a4d90cf03b656R239-R245). So I think that it may easily be the case.